src/common/search/SearchService.js

```diff
@@ -19,7 +19,7 @@
       searchlayer_ = new ol.layer.Vector({
         metadata: {
           title: $translate.instant('search_results'),
-          internalLayer: true
+          searchLayer: true
```

Comment:
Looks like it is actually referenced in the FeatureDiffService - adjusting code so this does not break anything.